README.md

```diff
@@ -6,6 +6,15 @@ This is a script that generates an interactive geo heatmap from your Google loca
 
 ## Getting Started
 
+### Using docker web-server
+```shell
+docker build -t geo_heatmap_web . 
+docker run -d -p 80:80 geo_heatmap_web
+```
+
+Open browser on `http://localhost`
+___
+
```

Comment:
Would be nice to have a clearer destinction between the two options (docker web server and running the script normally). 
Could you also add a short explanation to this options since it is a nice option for inexperienced users but they might not know what docker is or what it means to run a web server locally.

Reply:
"normally" sounds not quite right. Maybe ``Run the script without Docker`` or ``Run the script natively``?

Reply:
Done :smile: